circle.yml

```diff
@@ -1,3 +1,16 @@
 machine:
   node:
-    version: 6.5.0
+    version: 6.10.1
+dependencies:
```

Comment:
I was under the impression that yarn was officially supported by Circle implying that we do not need to do any setup ourselves.